docs/source/conf.py

```diff
@@ -1,72 +1,69 @@
 
-# Load local extensions (e.g. mermaid)
-sys.path.insert(0, str(Path.cwd().resolve() / "tools"))
+extensions = []
 
-# Add any Sphinx extension module names here, as strings. They can be
-# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
-# ones.
-extensions = ["mermaid", "mermaid_inheritance", "myst_parser", "breathe"]
-
-# Configuration of Breathe Doxygen interopt
-breathe_projects = {"libmamba": os.environ.get("MAMBA_DEV_DOXYGEN_XML_DIR", "../xml")}
-breathe_default_project = "libmamba"
-breathe_default_members = ("members", "undoc-members")
-breathe_show_include = False
-
-# Add any paths that contain templates here, relative to this directory.
 templates_path = ["_templates"]
+exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
 
-# List of patterns, relative to source directory, that match files and
-# directories to ignore when looking for source files.
-# This pattern also affects html_static_path and html_extra_path.
-exclude_patterns = []
-
+language = "en"
 
 # -- Options for HTML output -------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
+
+html_theme = "alabaster"
+html_static_path = ["_static"]
+
+# The file above was generated using sphinx 8.1.3 with this command:
+# sphinx-quickstart --project "mamba" --author "QuantStack & mamba contributors" -v "latest" -r "latest" -l en --no-sep --no-makefile --no-batchfile
+# These are custom options for this project
 
-# The theme to use for HTML and HTML Help pages.  See the documentation for
-# a list of builtin themes.
-#
 html_theme = "sphinx_book_theme"
 html_logo = "_static/logo.png"
 html_title = "documentation"
 
 html_theme_options = {
+    "path_to_docs": "docs/source",
+    "repository_branch": "main",
     "repository_url": "https://github.com/mamba-org/mamba",
+    "use_download_button": True,
+    "use_edit_page_button": True,
+    "use_issues_button": True,
     "use_repository_button": True,
 }
 
-# Add any paths that contain custom static files (such as style sheets) here,
-# relative to this directory. They are copied after the builtin static files,
-# so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ["_static"]
+import os  # noqa: E402
+import sys  # noqa: E402
+from pathlib import Path  # noqa: E402
+
+# Load local extensions (e.g. mermaid)
+sys.path.insert(0, str(Path.cwd().resolve() / "tools"))
+
+# Add Sphinx extension modules
+extensions = ["mermaid", "mermaid_inheritance", "myst_parser", "breathe"]
+
+# Configuration of Breathe Doxygen interopt
+breathe_projects = {"libmamba": os.environ.get("MAMBA_DEV_DOXYGEN_XML_DIR", "../xml")}
+breathe_default_project = "libmamba"
+breathe_default_members = ("members", "undoc-members")
+breathe_show_include = False
+
+# Configure links which should be ignored during linkcheck
+linkcheck_ignore = [
+    r".*github\.com.*#",  # javascript based anchors
```

Comment:
GitHub Readme files have javascript based anchors, which causes `linkcheck` errors. I also fixed 2 redirect warnings below.
Now, docs links are correct (and maybe I will add a CI step later)